### ISSUE TITLE
The pew shell config breaks my shell

### ISSUE THREAD

**Issue by bochecha** [berdario/pew#111]
I'm not sure where the problem comes from. It could be pew, it could be bash, maybe even vte, who knows?

All I know is that when I add these lines to my `~/.bashrc`, then my shell starts misbehaving:

```bash
if [ -f /usr/bin/pew ]; then
  source $(pew shell_config)
fi
```

The symptoms are that when using the arrow keys to navigate through older commands, eventually the beginning of the command line gets messed up.

Here is an example session:

* the lines above are in my `~/.bashrc`
* I start a new shell (and so it has sourced the `~/.bashrc`
* I press the `up` arrow, the command line shows the last command I entered, i.e `sudo dnf repoquery --whatprovides 'libwebkit2gtk-4.0.so.37()(64bit)'`
* I press the `up` arrow again, the command line now shows `sudorpm -q --requires epiphany`
    The command I really had entered was `rpm -q --requires epiphany`
* now I press and hold the `backspace` key, to erase the command line, and it clears everything, until the `sudo` word, which can never be cleaned (at that point if I use the `up` or `down` arrows to navigate the history again, that `sudo` word will always be there, prepended to the command displayed)
* if I now enter a command, say `ls` for example, then what I see is `sudols`, but what gets executed is just `ls`

So the display is completely broken, with the first "word" of a command remaining stuck at the beginning of a line.

It is extremely disturbing, as it breaks the mental flow of typing commands making you wonder why this word is there and how to remove it.

Another symptom is that long lines get wrapped badly:

![screenshot from 2016-07-11 11-04-54](https://cloud.githubusercontent.com/assets/272170/16725405/66a09ea6-4757-11e6-92d2-9c1a71f74d6e.png)

Instead, I would expect:

![screenshot from 2016-07-11 11-05-18](https://cloud.githubusercontent.com/assets/272170/16725414/6ff2678c-4757-11e6-8c9b-e50770b27f9e.png)

And I properly get the latter when I **don't** source the pew shell config.

As it is, it pretty much destroys my shell experience, and so I've reverted to not sourcing the pew shell config.

**Comment by xnegativx** [comment]
Had the same problem here, PS1 config is in init.bash is faulty. 
Fixed it.. then saw that PR was already submitted, see: 
https://github.com/berdario/pew/pull/95.

**Comment by ilourt** [comment]
Solve the problem for me too.  #95

**Comment by berdario** [comment]
I finally merged #95, so now this should be fixed

Sorry for the delay
